src/components/addSourceWizard/addSourceWizardStepThree.js

```diff
@@ -19,7 +19,10 @@ import { EMPTY_CONTEXT, translate } from '../i18n/i18n';
  * @returns {React.ReactNode}
  */
 const AddSourceWizardStepThree = ({ add, error, fulfilled, pending, name, t }) => (
-  <EmptyState className="quipucords-empty-state">
+  <EmptyState
+    className="quipucords-empty-state"
+    data-test-state={(error && 'error') || (pending && 'pending') || (fulfilled && 'fulfilled') }
+  >
```

Comment:
...looks like this code base could benefit from the data-test IDs applied through the translation tool

**fix:**
the linter is asking you to remove the space between the fulfilled condition and bracket 
```
(fulfilled && 'fulfilled')[space]}
```
**OR recommendation:**
what you have works, but...
Did you also try leveraging the existing conditions?

Starting around line `26` you could look at applying a `data-test` to each of the `EmptyStateIcon` components. You'd have to confirm, but on a light check it does appear `fulfilled` is coming through on the SVG, and `ouiaId` does kinda.

Another way around it is to repeat what you did may have done before and just apply a `<span data-test="[state]">` around a wrapping element.




```
{error && <EmptyStateIcon data-test="error" icon={ExclamationCircleIcon} color={red.value} />}
{fulfilled && <EmptyStateIcon data-test="fulfilled" icon={OutlinedCheckCircleIcon} color={green.value} />}
{pending && <EmptyStateIcon data-test="pending" icon={Spinner} />}
```

![Screen Shot 2023-08-21 at 10 11 22 AM](https://github.com/quipucords/quipucords-ui/assets/3761375/5dda6685-a394-4a57-be42-dd9195873727)

Reply:
I went straight to this first solution I had, didn't try anything else :) .

I tried adding attribute to icon now, and it works. Works like a charm on Camayoc side, too. Done that in 3bfc641f4261b005e1aea307d110d86aabd8600f